spec/requests/api/articles_index_spec.rb

```diff
@@ -2,14 +2,36 @@
   subject { response }
 
   describe 'when there are some article in the database' do
-    let!(:articles) { create_list(:article, 2) }
-    before do
-      get '/api/articles'
+    let(:tech) { create(:category, name: 'Tech') }
```

Comment:
naming issue. I think it is better to name this variable :category instead of :tech.

Reply:
This has been changed to category